packages/react-juce/src/lib/BlueprintBackend.ts

```diff
@@ -123,6 +168,21 @@ export class ViewInstance {
     // of a css prop, first convert it to kebab-case
     propKey = cssPropsMap[propKey] || propKey;
 
+    // convert provided color string to alpha-hex code for JUCE
+    if(propKey.includes("color")){
+      //converts any colorStrings in linear gradient to alpha-hex
+      if(value.includes("linear-gradient")) {
+        const lgValues = value.split(/\((.+)/)[1].split(/\)$/)[0].replace(/ /g, "").split(/,+(?![^\(]*\))/);
+        const lgValuesConverted = lgValues.map(value => {
+          return colorStringToAlphaHex(value);
+        })
+        value = "linear-gradient(" + lgValuesConverted.join(",") + ")"
```

Comment:
Yea exactly, you're on the right path here. My suggestion from above is basically just to take it one step further here before we pass it over to the native API, doing our start side + angle calculations in here.